src/app/features/create/components/hydrate-new-vehicle-record/hydrate-new-vehicle-record.component.ts

```diff
@@ -31,13 +39,51 @@ export class HydrateNewVehicleRecordComponent {
     );
   }
 
+  isInvalid: boolean = false;
+  vehicleType?: VehicleTypes;
```

Comment:
Please list variables and properties above the constructor.

Reply:
done, sorry